# FUS Protein Mammal Variation Viewer

### To View Spatial Variation of FUS protein follow this link:
##### https://achevann.github.io/MutationViewer/viewer.html

### Project: A number of substitutions in the fus protein (and other proteins with prion domains) that have been "linked to" disease  through clinical surveys, but may not necessarily be relevant to the disease phenotype. Here we analyze the orthologous fus protein sequences from mammals, to observe if these animals have any variation at each location within the human sequence. Variation at the points of interest in the sequence would indicate that mutations in amino acids at those locations may not be related to the disease phenotype.

### Description: This visualization displays amino acid variations found in publically available mammal fus protein sequences. This circular view of the variation data allows for a visual seqmentation of the location of high and low amounts of variation. A visual segmentation raises questions about local conservation of amino acid sequences and/or regional functionality in the progression of ALS.

### Layer ONE: Divides the primary structure of the fus protein into each amino acid location by regions within the protein.
### Layer TWO: The Amino acid single letter abbreviation and location within the primary structure.
### Layer THREE: The Amino acid substitutions found from clinical ALS data.

### When highlighting amino acid locations, the amino acid location as well as the number of mammals found with an amino acid variation at that location is displayed in the center of the visualization.

### Adapted from https://observablehq.com/@confessin/d3-zoomable-sunburst creaded with D3 Visualization Library.
